echidna/core/spectra.py

```diff
@@ -491,8 +491,21 @@ def scale(self, num_decays):
 
         Args:
           num_decays (float): Number of decays this spectra should represent.
+
+        .. warning:: Since :attr:`_num_decays` is set to equal the
+          :obj:`num_decays` supplied here, after calling :meth:`scale`,
+          the type of :attr:`_num_decays` will match the type of
+          :obj:`num_decays` supplied. E.g.::
+
+            >>> spectrum.scale(100)
+            >>> isinstance(spectrum._num_decays, int)  # True
+            >>> spectrum.scale(100.)
+            >>> isinstance(spectrum._num_decays, float)  # True
+
         """
-        self._data = numpy.multiply(self._data, num_decays / self._num_decays)
+        # Cast self._num_decays as float
+        self._data = numpy.multiply(self._data,
+                                    num_decays / float(self._num_decays))
         self._num_decays = num_decays
```

Comment:
In fact this line should be changed to 
``` python
self._num_decays = float(num_decays)
```
Just in case any other method uses num_decays.  This also negates the need for the warning on lines 494 - 504